lib/response.js

```diff
@@ -34,12 +34,19 @@ inherits(Response, EventEmitter);
  * @function redirect
  * @memberof module:nighthawk.Response
  * @instance
+ * @param {Number} statusCode - Optional status code discarded by nighthawk but handled for backend parity
  * @param {String} u - The new url
  */
-Response.prototype.redirect = function (u) {
+Response.prototype.redirect = function (statusCode, u) {
 	// In a timeout because page.js did it, but I believe it is better
 	// to let the current route handler finish its stuff before starting
 	// another round of routing, otherwise we could get some weird behavior.
+
+	// Sort out arguments by handling the optional statusCode
+	if (typeof statusCode === 'string') {
+		u = statusCode;
```

Comment:
ohh didn't realize we're even using status at all on the front end. figured we're just discarding it. i see it though